documentation/MeetingMinutes/MOM-2023-11-15.md

```diff
@@ -0,0 +1,125 @@
+  - PR #74 "Consent related use cases (former PR#33)". Related with issue #21.
+    - Set to `Draft`. It is on hold at the moment. No updates from last call.
+
+<br>
+
+## Issues and PRs
+
+| **Issue** | Owner | Description |
+| ----------| ----- | ----------- |
+| **Closed issues** |  |  |
+| [#32](https://github.com/camaraproject/IdentityAndConsentManagement/issues/32) ([~~PR#75~~](https://github.com/camaraproject/IdentityAndConsentManagement/pull/75)) | Orange | ~~**The concept of purpose**~~<br>As agreed in the last WG call (18/10), PR #75 has been merged and issue #32 closed. |
+| [#59](https://github.com/camaraproject/IdentityAndConsentManagement/issues/59) ([~~PR#75~~](https://github.com/camaraproject/IdentityAndConsentManagement/pull/75)) | Orange | ~~**documentation/CAMARA-API-access-and-user-consent.md diagram update purpose/scope**~~<br>Fixed by PR #75. |
+| [#67](https://github.com/camaraproject/IdentityAndConsentManagement/issues/67) ([~~PR#68~~](https://github.com/camaraproject/IdentityAndConsentManagement/pull/68)) | Deutsche Telekom AG | ~~**Make the technical ruleset description precise.**~~<br>Fixed by PR #68. |
+| [#80](https://github.com/camaraproject/IdentityAndConsentManagement/issues/80) | GSMA | ~~**Can the use of purpose in authorisation request be considered stable?**~~<br>`CAMARA-API-access-and-user-consent.md` document is the result of the short term agreement reached on Issue #32 and must be considered the current stable position on what refers to purpose declaration in the authorization process. |
+| **New issues** |  |  |
+| [#78](https://github.com/camaraproject/IdentityAndConsentManagement/issues/78) | Orange | **clarify the "credentials" notion in CIBA workflow authentication**<br> Issue opened by Orange to specify the client authentication that must be supported in CAMARA. (client_secret, private_key_jwt).<br>- Ludovic (Orange) comments that in the Spanish champion this issue was raised and some telcos have one auth mechanism and others have another different mechanism and it may be that the partners/aggregators only want to have one or can only work with one.<br>- Jesús (Telefónica) points out that these kinds of questions, which are open in CAMARA (and also in GSMA), actually may point to a need that has not yet been met. It looks like there may be a need for an Open Gateway / CAMARA profile/binding documenting what are the mandatory parameters in the authorisation requests, what are the error codes and error scenarios, what is the client authentication supported, what are the login hint methods supported for CIBA, etc... And this profile/binding should be in one place to avoid having to maintain more than reference documentation, as also reported in issue #82.<br>- Axel (DT) comments on work in Financial APIs - https://openid.net/certification/#FAPI-CIBA_OPs<br>- Eric (Vodafone) comments that he is in favour of defining a profile but that it would only apply to the final consumer of the API. |
```

Comment:
If I got it right the sentence should be:
 _And this profile/binding should be in one place to avoid having to maintain more than **one** reference documentation,_

```suggestion
| [#78](https://github.com/camaraproject/IdentityAndConsentManagement/issues/78) | Orange | **clarify the "credentials" notion in CIBA workflow authentication**<br> Issue opened by Orange to specify the client authentication that must be supported in CAMARA. (client_secret, private_key_jwt).<br>- Ludovic (Orange) comments that in the Spanish champion this issue was raised and some telcos have one auth mechanism and others have another different mechanism and it may be that the partners/aggregators only want to have one or can only work with one.<br>- Jesús (Telefónica) points out that these kinds of questions, which are open in CAMARA (and also in GSMA), actually may point to a need that has not yet been met. It looks like there may be a need for an Open Gateway / CAMARA profile/binding documenting what are the mandatory parameters in the authorisation requests, what are the error codes and error scenarios, what is the client authentication supported, what are the login hint methods supported for CIBA, etc... And this profile/binding should be in one place to avoid having to maintain more than one reference documentation, as also reported in issue #82.<br>- Axel (DT) comments on work in Financial APIs - https://openid.net/certification/#FAPI-CIBA_OPs<br>- Eric (Vodafone) comments that he is in favour of defining a profile but that it would only apply to the final consumer of the API. |
```

Reply:
I can help provide insights on how this can be solved and address each persons concerns. 

Can I have 24-hrs to come back with a suggestion? This issue is exactly the goal of a “Consent and Measurement API” would want to solve.

Would that be okay?

Reply:
>I can help provide insights on how this can be solved and address each persons concerns.
>
>Can I have 24-hrs to come back with a suggestion? This issue is exactly the goal of a “Consent and Measurement API” would want to solve.
>
>Would that be okay?

@nickvenezia This PR is just to merge the meeting notes from yesterday's call. If you have any suggestions or proposals, please provide them by creating a new issue in the github project as suggested yesterday, so that all WG participants can review it and provide feedback if necessary.

Reply:
@rartych Should be fixed by https://github.com/camaraproject/IdentityAndConsentManagement/pull/83/commits/a704b9df318d471f04e6b26c06ee994418f55997